packages/node/src/integrations/node-fetch.ts

```diff
@@ -46,32 +35,13 @@ const _nativeNodeFetchIntegration = ((options: NodeFetchOptions = {}) => {
   const _breadcrumbs = typeof options.breadcrumbs === 'undefined' ? true : options.breadcrumbs;
   const _ignoreOutgoingRequests = options.ignoreOutgoingRequests;
 
-  async function getInstrumentation(): Promise<FetchInstrumentation | void> {
-    // Only add NodeFetch if Node >= 18, as previous versions do not support it
-    if (NODE_MAJOR < 18) {
-      DEBUG_BUILD && logger.log('NodeFetch is not supported on Node < 18, skipping instrumentation...');
```

Comment:
Do we not need to guard here anymore at all? I guess this just does nothing on node 14 then?

Reply:
`diagnostic_channel` is available from Node v14.17.0 there just wont be an events on the channel until later node versions

Reply:
nice, makes sense!